client/dashboard/me/billing-purchases/cancel-purchase/cancel-purchase-form/index.tsx

```diff
@@ -436,12 +440,12 @@ export default function CancelPurchaseForm( providedProps: CancelPurchaseFormPro
 						className="jetpack-cancellation-offer__accept-cta"
 						variant="primary"
 						onClick={ () => {
-							onClickAccept();
+							onClickAccept && onClickAccept();
 						} }
-						disabled={ isApplyingOffer ?? ( false || offerApplyError ) ?? false }
-						isBusy={ isApplyingOffer }
+						disabled={ isApplyingOffer ?? ( false || Boolean( offerApplyError ) ) ?? false }
+						isBusy={ isApplyingOffer ?? false }
 					>
-						{ isApplyingOffer ? __( 'Getting Discount' ) : __( 'Get discount' ) }
+						{ isApplyingOffer ?? false ? __( 'Getting Discount' ) : __( 'Get discount' ) }
```

Comment:
I _think_ this is unnecessary? If `isApplyingOffer` is undefined, it should still trigger the falsy branch of the ternary without the null coalesce being needed.

Reply:
This solution appeased the type checker at one point, but no longer seems to need it. I've removed it.